packages/calypso-e2e/src/rest-api-client.ts

```diff
@@ -628,4 +656,36 @@ export class RestAPIClient {
 
 		return response;
 	}
+
+	/* Shopping Cart */
+
+	/**
+	 * Clears the shopping cart.
+	 *
+	 * @param {number} siteID Site that has the shopping cart.
+	 * @throws {Error} If the user doesn't have access to the siteID.
+	 * @returns {{success:true}} If the request was successful.
```

Comment:
Similar reason here as `setMySettings` - if the call successfully clears the shopping cart, the only return value is {success:true}.